src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,43 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
+        # cur_index = i
+        # smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
+        # (hint, can do in 3 loc)
+        for j in range(i, len(arr) - 1):
+            if arr[i] > arr[j + 1]:
+                # TO-DO: swap
+                arr[i], arr[j + 1] = arr[j + 1], arr[i]
 
+    return arr
 
 
-        # TO-DO: swap
-
-
+# print(selection_sort([0, 1, 10, 3, 4, 5, 6, 8, 9, 7]))
+# TO-DO:  implement the Bubble Sort function below
 
 
+def bubble_sort(arr):
```

Comment:
Nice work! I like the way you handled moving through the loop. It's like a post-modern for loop hehe. It seems you have a solid understanding of how to use a while loop so you should have fun as we move into depth and breadth first search.